package.json

```diff
@@ -15,11 +15,11 @@
     "dev": "concurrently -rk 'npm run build:watch' 'npm run lint:watch'",
     "lint": "eslint .",
     "lint:fix": "npm run lint --silent -- --fix",
-    "lint:watch": "watch 'npm run lint --silent' src",
+    "lint:watch": "watch 'npm run lint --silent' src test",
     "prepublish": "npm run build",
-    "pretest": "npm run lint",
-    "test": "ava",
-    "test:watch": "watch 'npm test --silent' src test"
+    "posttest": "npm run lint --silent",
+    "test": "mocha",
+    "test:watch": "concurrently -rk 'npm run test --silent -- -w' 'npm run lint:watch'"
```

Comment:
Moved lint to `posttest` so the lint results show after the tests run.  Helps keep linter happy while testing.